src/WebWorker.ts

```diff
@@ -30,12 +30,33 @@ type ExceptionThrownCallback = (
 ) => void;
 type JSHandleFactory = (obj: Protocol.Runtime.RemoteObject) => JSHandle;
 
+/**
+ * The WebWorker class represents a [WebWorker](https://developer.mozilla.org/en-US/docs/Web/API/Web_Workers_API).
```

Comment:
these need to use the TSDoc @link syntax: https://api-extractor.com/pages/tsdoc/tag_link/